tests/python/pants_test/backend/python/rules/test_python_test_runner_integration.py

```diff
@@ -48,7 +48,7 @@ def run_passing_pants_test(self, trailing_args):
   def run_failing_pants_test(self, trailing_args):
     pants_run = self.run_pants(trailing_args)
     self.assert_failure(pants_run)
-    self.assertEqual('Tests failed\n', pants_run.stderr_data)
+    self.assertThat('Tests failed\n' in pants_run.stderr_data)
```

Comment:
Is this change related?

Reply:
yes, similar here - the error output is now changed to include the `deprecated:hint_message`, hence the `in`